mods/cnc/maps/gdi03/map.yaml

```diff
@@ -823,15 +823,35 @@ Rules:
 			Amount: -10
 		-Capturable:
 	HQ:
+		Buildable:
+			Prerequisites: ~disabled
+	NOHQ:
+		Inherits: HQ
+		Buildable:
+			Prerequisites: proc
+		Tooltip:
+			Description: Provides an overview of the battlefield.\n  Requires power to operate.
+		-AirstrikePower:
+		RenderBuilding:
+			Image: hq
+	airstrike.proxy:
```

Comment:
We are probably putting more thinking in this than Westwood did so I will just merge this now.